yellowbrick/features/importances.py

```diff
@@ -122,6 +124,10 @@ def fit(self, X, y=None, **kwargs):
         # Get the feature importances from the model
         self.feature_importances_ = self._find_importances_param()
 
+        # Check if feature importances is a multidim array & if so flatten
+        if self.feature_importances_.ndim > 1:
+            self.feature_importances_ = np.mean(self.feature_importances_, axis=0)
```

Comment:
Just to record the discussion from Slack in the issue: we're expecting `LogisticRegression` to have a `coefs_` property whose shape is [`(n_classes, n_features)`](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/linear_model/logistic.py#L1100) therefore by taking the column-wise mean (e.g. axis=0) we're getting the average coefficient of all classes for each feature and using that to plot importances.

For future us,  would you mind updating the comment as follows?

```python
# If feature importances is a multidim array, we're expecting a shape of 
# (n_classes, n_features) therefore we flatten by taking the average by 
# column to get shape (n_features,)  (see LogisticRegression) 
```

Additionally, in the future I could see a very interesting feature importance visualizer that uses this shape of array to create a stacked bar chart of feature importances for each class/feature combination. Would you mind opening an issue for that or would you like me to?